README.rst

```diff
@@ -116,6 +116,23 @@ takes the last part.  Some exceptions, most for subversion:
 
 - Git checkouts lose the ".git" at the end of the url.
 
+Optionally, a "preserve_tree" option may be set for each group in the
+configuration. The contents of the "preserve_tree" option should be a list
+of strings. If the "preserve_tree" option is set, then the default
```

Comment:
I wouldn't emphasise the "list"-ness too much. And "list" and "string" are so datatype-like. What about: "In `preserve_tree` you can set one or more base checkout urls."?

Ah well, leave that in for now. I'll make it a bit shorter myself, I've got some ideas :-) Before making a release I'll let you see it if you think it is still clear enough.